src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,127 @@
+import { useState } from "react";
+
+import Scorad from "../Scorad/Scorad";
+import Button from "../Button/Button";
+import Section from "../Section/Section";
+
+import "./Dashboard.css";
+
+import { addingFormData } from "../../constants/dashboardInputs";
+
+function Dashboard() {
+  const [todayScorad, setTodayScorad] = useState(null);
+  const [displayForm, setDisplayForm] = useState(false);
+  const [drugs, setDrugs] = useState([]);
+  const [cares, setCares] = useState([]);
+  const [events, setEvents] = useState([]);
+  const [notes, setNotes] = useState([]);
+  const [listItem, setListItem] = useState({});
+
+  function handleButtonClick(e) {
+    if (e.target.name === "scorad") {
+      setDisplayForm(true);
+    }
+  }
+
+  function handleInputChange(e) {
+    const name = e.target.name;
+    const value = e.target.value;
+    setListItem((values) => ({ ...values, [name]: value }));
+  }
+
+  function handleDrugAdding(e) {
+    e.preventDefault();
+    let newDrugs = [...drugs];
+    newDrugs.push(listItem);
+    console.log(newDrugs);
+    setDrugs(newDrugs);
+    setListItem({});
+    e.target.reset();
```

Comment:
It should be just one line of code.
```suggestion
    setDrugs([...drugs, newDrug])
```